src/modules/vax/VaxLocationDetail.tsx

```diff
@@ -77,10 +80,16 @@ export default function VaxLocationDetail({ loading, isUserLocationExist, locati
 
     return null;
   };
-
   return (
     <>
-      <Flex alignItems="center" flexDirection="row" justifyContent="space-between" p={4}>
+      <Flex
+        alignItems="center"
+        flexDirection="row"
+        justifyContent="space-between"
+        onClick={onOpen}
```

Comment:
**Never** use `onClick` on non-interactable elements (e.g. `div`). `chakra-ui` has a [LinkOverlay component](https://chakra-ui.com/docs/navigation/link-overlay) for a more accessible alternative.